packages/core/src/com/communication.ts

```diff
@@ -941,6 +938,7 @@ export class Communication {
                 this.pendingCallbacks.delete(callbackId);
                 delete this.callbacks[callbackId];
                 clearTimeout(timerId);
+                error.message += `\nCaused by: ${JSON.stringify(cleanMessageForLog(message), null, 2)}`;
```

Comment:
@barak007 @idoros 

These are error messages from cross-environment calls, they're shown to the users, and now they look like this:

<img width="524" alt="image" src="https://github.com/wixplosives/engine/assets/415252/e229a123-75d5-438c-9910-38a359451763">



I'm removing this, it's not acceptable. There's a comment in this very file about the importance of not tampering error messages:
https://github.com/wixplosives/engine/blob/80d420d3b91d0170650b090adf4d6c3203094a14/packages/core/src/com/communication.ts#L869-L871